### DISCUSSION TITLE
2008/01/07/%E5%BD%A2%E7%8A%B6%E7%9F%A5%E8%A7%89%E4%B8%AD%E7%9A%84%E5%88%86%E7%BB%84

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#204] [Announcements]
# 2008/01/07/%E5%BD%A2%E7%8A%B6%E7%9F%A5%E8%A7%89%E4%B8%AD%E7%9A%84%E5%88%86%E7%BB%84

昨天看《心理学》恰好看到知觉这一章中的视觉形状知觉中的“分组”一节。这一节介绍的知识对我们程序界面设计颇有启发作用。它还为我以前总结的一些界面设计的经验提供了理论依据。

https://blog.qizhen.xyz/2008/01/07/%E5%BD%A2%E7%8A%B6%E7%9F%A5%E8%A7%89%E4%B8%AD%E7%9A%84%E5%88%86%E7%BB%84

**Comment by ruanqizhen** [comment]
LIVIA
2008年02月26日 18:45 编辑
好神奇~

回复
竹贤
2008年01月11日 08:15 编辑
看来“眼见为实”的说法也有局限性啊。

回复
鱼
2008年01月8日 13:17 编辑
是啊，很专业

回复
WL
2008年01月8日 04:23 编辑
有道理啊

回复
acca
2008年01月8日 00:17 编辑
突然发现这里的文章毛专业…
略作了解也不错~

回复
竹贤
2008年01月7日 21:39 编辑
确实如此。
